components/raftstore/src/store/worker/metrics.rs

```diff
@@ -44,14 +47,54 @@ make_static_metric! {
         epoch,
         applied_term,
         channel_full,
+        cache_miss,
         safe_ts,
     }
 
-    pub struct ReadRejectCounter : IntCounter {
-       "reason" => RejectReason
+    pub struct LocalReadRejectCounter : LocalIntCounter {
+       "reason" => RejectReason,
     }
 }
 
+pub struct LocalReadMetrics {
```

Comment:
Why some fields of `ReadMetrics` are not in `LocalReadMetrics`?

Reply:
they are sub-fields in `reject_reason`